### PR TITLE
feat: output: show key-value pairs in terminal if verbosity > 0

### PR DESCRIPTION
**Pull request by floridoo**
Jira: https://jira.d2iq.com/browse/D2IQ-85586

### CODE REVIEW

**Comment by mhrabovcin** [comment]
We should update https://github.com/mesosphere/dkp-cli-runtime/blob/main/core/output/output.go#L96 to tell that values are printed when verbosity level > 0.

File: core/output/interactive.go [hunk 1]
```diff
@@ -21,18 +21,24 @@ func NewInteractiveShell(out, errOut io.Writer, verbosity int) Output {
...
 type interactiveShellOutput struct {
-	out       io.Writer
-	errOut    *spinner
-	verbosity int
-	status    string
-	lock      sync.Mutex
+	out           io.Writer
+	errOut        *spinner
+	verbosity     int
+	level         int
```
Comment (mhrabovcin) [inline core/output/interactive.go:32]: Why are we introducing new `level` variable? Couldn't we use `verbosity` here?

Comment (floridoo) [reply core/output/interactive.go:32]: Verbosity is something different: it specifies how verbose the output is (maximum level that will be printed to output)

Comment (mhrabovcin) [reply core/output/interactive.go:34]: So `verbosity` represents the maximum verbosity that will print something and `level` is the current verbosity value?

Comment (mhrabovcin) [reply core/output/interactive.go:34]: I see the new comments, thanks!

Comment (floridoo) [reply core/output/interactive.go:34]: Yes, or to be more precise: `verbosity` is the output's (=logger's) verbosity. `level` is the verbosity of the messages you want to output.

**Review by mhrabovcin — APPROVED** [review]

**Pull request merged**